src/main.js

```diff
@@ -7,34 +7,32 @@ var inputTitle = document.querySelector('#input-title')
 var inputBody = document.querySelector('#input-body')
 var userIdeas = document.querySelector('.user-ideas')
 var ideaCardTemplate = document.querySelector('#unique-card')
+var searchBarInput = document.querySelector('#input-search')
 
 var saveButton = document.querySelector('#button-save')
 var deleteIdea = document.querySelector('.icon-delete')
-var favIdea = document.querySelector('.icon-star')
 var showStarredIdeasButton = document.querySelector('#button-starred')
 
 // event listeners
 window.addEventListener('load', displayStoredIdeas)
 saveButton.addEventListener('click', saveNewIdea)
 inputBody.addEventListener('keyup', setSaveButtonState)
 inputTitle.addEventListener('keyup', setSaveButtonState)
-showStarredIdeasButton.addEventListener('click', toggleIdeasDisplay)
+showStarredIdeasButton.addEventListener('click', toggleStarredIdeasDisplay)
+searchBarInput.addEventListener('keyup', searchIdeas)
+
 
 userIdeas.addEventListener('click', function(event) {
   if (event.target.className === 'icon-delete') {
```

Comment:
Using the classnames here is ok, but a [data-attribute](https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes) would make sure that changes to classnames doesn't mess w/ functionality.